src/feature_scaling.jl

```diff
@@ -1,55 +1,130 @@
 """
-`μ = center!(X[, μ])`
+`μ = center!(X, obsdim[, μ])`
 
-Centers each row of `X` around the corresponding entry in the vector `μ`.
+Centers `X` along obsdim around the corresponding entry in the vector `μ`.
 If `μ` is not specified then it defaults to `mean(X, 2)`.
 """
-function center!{T<:AbstractFloat}(X::AbstractMatrix{T}, μ::AbstractVector = vec(mean(X, 2)))
-    k, n = size(X)
-    @inbounds for j = 1:k
-        for i = 1:n
-            X[j, i] = X[j, i] - μ[j]
-        end
+function center!(X, mu; obsdim=LearnBase.default_obsdim(X))
+  center!(X, mu, LearnBase.obs_dim(obsdim))
```

Comment:
the current default indentation for Julia is 4 spaces. so each indentation level should be 4 spaces more